packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -678,4 +751,55 @@ - (void)setMixWithOthers:(FLTMixWithOthersMessage *)input
   }
 }
 
+- (nullable NSNumber *)isPictureInPictureSupported:
+    (FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  return @([AVPictureInPictureController isPictureInPictureSupported]);
```

Comment:
If this is true, but `doesInfoPlistSupportPictureInPicture` returns NO, then plugin clients will be told the feature works at runtime, but any attempt to use it will immediately throw an error, which seems like very confusing behavior.

Reply:
Fair point, I will add that